### DISCUSSION TITLE
In BEX applications using Webpack, support for manifest V3

### DISCUSSION THREAD

**Discussion by thomas66777** [quasarframework/quasar#17404] [Ideas / Proposals]
I have a chrome extension written in quasar webpack manifest v2.  Chrome is beginning to display warnings for apps that are still using v2.  I can see that you have an option to use v3 through Vite but that is creating a lot of issues for me.  Do you plan on supporting v3 with Webpack? Thank you

![image](https://github.com/user-attachments/assets/bdc9c176-f029-4437-b4cf-ef1f5488a3d4)

**Comment by yusufkandemir** [comment]
It's supported in app-webpack v4 which is currently in beta. It won't be ported back to app-webpack v3 as it's too complicated to do so.
Release notes: [`@quasar/app-webpack v4.0.0-beta.1`](https://github.com/quasarframework/quasar/releases/tag/%40quasar%2Fapp-webpack-v4.0.0-beta.1)
Discussion thread: https://github.com/quasarframework/quasar/discussions/16877
